contrib/buildahimage/upstream/Containerfile

```diff
@@ -17,7 +17,10 @@ ENV GOPATH=/root/buildah
 # to the container.
 # Finally remove the buildah directory and a few other packages
 # that are needed for building but not running Buildah
-RUN useradd build; yum -y update;  rpm --restore shadow-utils 2>/dev/null; yum -y install --enablerepo=updates-testing \
+RUN useradd build; \
+    dnf -y update;  && \
+    rpm --setcaps shadow-utils 2>/dev/null && \
+    dnf -y install --enablerepo=updates-testing \
```

Comment:
But...but...but....this is Fedora...`dnf` is clearly superior :grin:  Otherwise why would `yum` simply be a symlink to `dnf`?

Seriously, use `dnf` it's more portable (and better for our copy-paste).

Reply:
dnf is everywhere we care about at this point IE Distributions equal or newer then RHEL8.